src/main/java/org/sasanlabs/service/impl/EndPointsInformationProvider.java

```diff
@@ -235,10 +225,26 @@ private void addFacadeResourceInformation(
                         for (AttackVector attackVector : attackVectors) {
                             List<VulnerabilityType> facadeLevelVulnerabilityTypes =
                                     new ArrayList<VulnerabilityType>();
+                            org.sasanlabs.vulnerability.types.VulnerabilityType[]
+                                    vulnerabilityTypes = attackVector.vulnerabilityExposed();
                             for (org.sasanlabs.vulnerability.types.VulnerabilityType
                                     vulnerabilityType : vulnerabilityTypes) {
                                 facadeLevelVulnerabilityTypes.add(
                                         new VulnerabilityType("Custom", vulnerabilityType.name()));
+                                facadeVulnerabilityTypes.add(
```

Comment:
don't add the CWE and WASC if they are having null value. Also can we avoid ternary operator?